examples/gen_test_data/gen_test_data/generate_test_data_flow/flow.dag.yaml

```diff
@@ -0,0 +1,153 @@
+  type: prompt
+  source:
+    type: code
+    path: generate_suggested_answer_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+    question: ${validate_test_question.output}
+  use_variants: false
+- name: validate_and_generate_seed_question
+  type: python
+  source:
+    type: code
+    path: validate_and_generate_seed_question.py
+  inputs:
+    connection:
+    model_or_deployment_name: gpt-4
+    score_text_trunk_prompt: ${score_text_trunk_prompt.output}
+    seed_question_prompt: ${seed_question_prompt.output}
+    context: ${inputs.text_chunk}
+    score_threshold: 4
+    response_format: text
+    temperature: 1
+    max_tokens: 512
+  use_variants: false
+- name: validate_and_generate_test_question
+  type: python
+  source:
+    type: code
+    path: validate_and_generate_test_question.py
+  inputs:
+    connection:
+    model_or_deployment_name: gpt-4
+    validate_seed_question_prompt: ${validate_seed_question_prompt.output}
+    seed_question: ${validate_and_generate_seed_question.output.question}
+    response_format: text
+    temperature: 1
+    max_tokens: 512
+  use_variants: false
+- name: validate_question_prompt
+  type: prompt
+  source:
+    type: code
+    path: validate_question_prompt.jinja2
+  inputs:
+    question: ${validate_and_generate_test_question.output.question}
+  use_variants: false
+- name: validate_test_question
+  type: python
+  source:
+    type: code
+    path: validate_test_question.py
+  inputs:
+    connection:
+    model_or_deployment_name: gpt-4
+    question_info: ${validate_and_generate_test_question.output}
+    validate_question_prompt: ${validate_question_prompt.output}
+    temperature: 1
+    max_tokens: 512
+  use_variants: false
+- name: generate_suggested_answer
+  type: python
+  source:
+    type: code
+    path: generate_suggested_answer.py
+  inputs:
+    connection:
+    context: ${inputs.text_chunk}
+    generate_suggested_answer_prompt: ${generate_suggested_answer_prompt.output}
+    question: ${validate_test_question.output}
+    model_or_deployment_name: gpt-4
+    temperature: 1
+    max_tokens: 512
+  use_variants: false
+- name: generate_debug_info
+  type: python
+  source:
+    type: code
+    path: generate_debug_info.py
+  inputs:
+    question_type: ${validate_and_generate_test_question.output.question_type}
+    text_trunk: ${inputs.text_chunk}
+    text_meta: ${inputs.text_meta}
+    validate_and_generate_seed_question_output: ${validate_and_generate_seed_question.output}
+    validate_and_generate_test_question_output: ${validate_and_generate_test_question.output}
+    validate_suggested_answer_output: ${validate_suggested_answer.output}
+- name: validate_suggested_answer_prompt
+  type: prompt
+  source:
+    type: code
+    path: validate_suggested_answer_prompt.jinja2
+  inputs:
+    answer: ${generate_suggested_answer.output}
+- name: validate_suggested_answer
+  type: python
+  source:
+    type: code
+    path: validate_suggested_answer.py
```

Comment:
Consistent with `question` and `text_trunk`, merge `validate` and `generate` to one python node.